cloud/store/store.go

```diff
@@ -78,8 +78,13 @@ type Indexer interface {
 	Close(context.Context) error
 }
 
+// Parser parses concrete class fields after deserialization
 type Parser interface {
+	// ParseClassUpdate parses a class after unmarshaling by setting concrete types for the fields
```

Comment:
**suggestion (if-minor):** 

```suggestion
        // ParseClass parses new updates by providing the current class data.
	ParseClass(class *models.Class) error
	
	// ParseClassUpdate parses a class after unmarshaling by setting concrete types for the fields
	ParseClassUpdate(class, update *models.Class) (*models.Class, error)
```

Reply:
Thanks for pointing this out. I will take care of it in next PR